openshift/core.app.yaml

```diff
@@ -105,6 +105,11 @@ objects:
               configMapKeyRef:
                 name: core
                 key: openshift.tenant.masterurl
+          - name: ALMIGHTY_FEATURE_WORKITEM_REMOTE
```

Comment:
We need to add this property to the config maps before merging it. Otherwise the build will fail